test/semantics/c_f_pointer.f90

```diff
@@ -0,0 +1,30 @@
+! Enforce 18.2.3.3
+
+include '../../module/__fortran_builtins.f90'
+include '../../module/iso_c_binding.f90'
```

Comment:
These includes aren't necessary. The error tests are run with a `-I` option that points to the predefined modules.

Reply:
Ok.